services/src/main/scala/org/apache/gearpump/services/MasterService.scala

```diff
@@ -26,7 +33,83 @@ trait MasterService extends HttpService {
             case Failure(ex) => complete(StatusCodes.InternalServerError, s"An error occurred: ${ex.getMessage}")
           }
         }
+      } ~
+      path("appmasters") {
+        onComplete((master ? AppMastersDataRequest).asInstanceOf[Future[AppMastersData]]) {
```

Comment:
@huafengw 

This is part of the legacy code moved from another file. I will start to clean this out after this PR.